test/transformer.test.js

```diff
@@ -1,123 +1,123 @@
 'use strict'
 
-const { test, before, teardown } = require('tap')
+const { test, describe, after } = require('node:test')
+const assert = require('assert')
 const Redis = require('ioredis')
 
 const createStorage = require('../src/storage')
 const { Cache } = require('../')
 
-let redisClient
-before(async (t) => {
```

Comment:
please keep before and after

Reply:
I added the `before` instruction to initialize Redis, which was missing, and encapsulated them within the test suite

Reply:
Added in the suite